operator/pkg/central/reconciler/reconciler.go

```diff
@@ -42,9 +45,30 @@ func RegisterNewReconciler(mgr ctrl.Manager) error {
 		pkgReconciler.WithReconcilePeriod(extensions.InitBundleReconcilePeriod),
 		pkgReconciler.WithPauseReconcileAnnotation(pauseReconcileAnnotation),
 	}
+
+	opts, err := addSelectorOptionIfNeeded(selector, opts)
+	if err != nil {
+		return err
+	}
+
 	return reconciler.SetupReconcilerWithManager(
 		mgr, platform.CentralGVK, image.CentralServicesChartPrefix,
 		proxy.InjectProxyEnvVars(translation.Translator{}, proxyEnv),
 		opts...,
 	)
 }
+
+func addSelectorOptionIfNeeded(selector string, opts []pkgReconciler.Option) ([]pkgReconciler.Option, error) {
+	if len(selector) == 0 {
+		return opts, nil
+	}
+	labelSelector, err := v1.ParseToLabelSelector(selector)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to parse central label selector")
```

Comment:
Just a thought, feel free to ignore: perhaps we should hint somehow that this comes from an environment variable? How about this?

```suggestion
		return nil, errors.Wrap(err, "failed to parse $CENTRAL_LABEL_SELECTOR")
```

The downside is that the actual variable name is specified in a different package, and might drift :-/

Reply:
I've added a log entry when the operator starts to indicate that the label selector has been obtained from an environment variable